moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryLabel.js

```diff
@@ -24,6 +24,7 @@ const ProjectSummaryLabel = ({
       <Typography
         className={className ?? classes.fieldLabelText}
         onClick={onClickEdit}
+        component="span"
```

Comment:
This fixed a console warning that kept popping up when the project summary view loaded - `<div> cannot appear as a child of <tr>.
`

Reply:
🙌